### PR TITLE
Update branch names for EN

### PR DESCRIPTION
**Pull request by Stinggyray**
Changes:

- `data/traits.json` has been deleted, and the traits have been consolidated with the branch translations into a `branches.json`, with the following schema:
```js
"subprofessionId": {
  "subclassName": "Name"
  "trait": "Does nothing"
}
```
- The `TRAIT_OVERRIDES` in `copy-gamedata.ts` have been updated to match the new EN translations better
- `globals.json` has been changed to no longer use the old bigass lookup table; instead, it imports all the data from `branches.json` (meaning after we update the game data, we don't need to manually change branch names in `global.json` anymore)

### CODE REVIEW

File: src/data/branches.json [hunk 1]
```diff
@@ -0,0 +1,210 @@
+{
+  "fortress": {
+    "subclassName": "Fortress",
+    "trait": "Prioritize <span class=\"keyword\">Long range splash attack</span> (splash radius of <span class=\"keyword\">1.0</span> tiles) when not blocking"
+  },
+  "wandermedic": {
+    "subclassName": "Wandering",
```
Comment (iansjk) [inline src/data/branches.json:7]: Should we perhaps use `branchName`?

File: src/scripts/copy-gamedata.ts [hunk 2]
```diff
@@ -54,20 +55,30 @@ const NAME_OVERRIDES: Record<string, string> = {
...
+  fortress: "Fortress",
+  wandermedic: "Wandering",
+  craftsman: "Artificer",
+};
 const TRAIT_OVERRIDES: Record<string, string> = {
   musha:
     "Can't be healed by other units. Recovers <@ba.kw>30/50/70</> (scales with elite promotion) self HP every time this operator attacks an enemy",
   chain:
-    'Attacks deal <@ba.kw>Arts damage</> and jump between <@ba.kw>3</> (<@ba.kw>4</> at Elite 2) enemies (jump range is <@ba.kw>1.7</> tiles). Each jump deals 20% less damage and inflicts an <@ba.kw>80%</> <span class="skill-tooltip">Slow</span> for <@ba.kw>0.5</> seconds',
+    'Attacks deal <@ba.kw>Arts damage</> and jump between <@ba.kw>3</> (<@ba.kw>4</> at Elite 2) enemies (jump range is <@ba.kw>1.7</> tiles). Each jump deals 15% less damage and inflicts an <@ba.kw>80%</> <span class="skill-tooltip">Slow</span> for <@ba.kw>0.5</> seconds',
```
Comment (iansjk) [inline src/scripts/copy-gamedata.ts]: Was this a mistake on our part?

**Review by iansjk — APPROVED** [review]
Apologies for the enormous PR due to the portrait filenames changing. In any case, this looks good to me, and I see that Chromatic picked up changes to the branch names.

Left some non-blocking comments to consider.

File: src/data/branches.json (hunk 1 above)
Comment (Stinggyray) [reply src/data/branches.json:7]: yes we should

File: src/pages/operators/index.tsx [hunk 3]
```diff
@@ -483,10 +483,16 @@ const Operators: React.VFC<Props> = ({ data }) => {
...
-                        )!.analysis.childMarkdownRemark.html,
+                        __html: operatorSubclasses
+                          .find(
+                            ({ subProfessionId }) =>
+                              subProfessionId === selectedSubProfessionId
+                          )!
+                          .analysis.childMarkdownRemark.html.replace(
+                            "BRANCHNAME",
+                            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                            `<strong>${selectedSubclass!} ${selectedClass!}s</strong>`
```
Comment (iansjk) [inline src/pages/operators/index.tsx:494]: Extra `s`? (`$selectedClass!}s`)

**Pull request merged**